### PR TITLE
Getting started docs

### PR DESCRIPTION
**Pull request by tim775**
## Hey, I just made a Pull Request!

I was working through the Getting Started tutorial and noticed a couple of inconsistencies that could be corrected.

#### :heavy_check_mark: Checklist

- ~~[ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))~~
- [x] Added or updated documentation
- ~~[ ] Tests for new functionality and regression tests for bug fixes~~
- ~~[ ] Screenshots attached (for UI changes)~~
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/getting-started/configuration.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ for 7 days, it's a lucky number.
   <img src='../assets/getting-started/gh-pat.png' alt='Screenshot of the GitHub Personal Access Token creation page' />
 </p>
 
-Set the scope to your likings. For this tutorial, selecting "repo" should be
+Set the scope to your likings. For this tutorial, selecting "repo" and "workflow" should be
```
Comment (jhaals) [inline docs/getting-started/configuration.md:221]:
```suggestion
Set the scope to your likings. For this tutorial, selecting "repo" and "workflow" is required as the scaffolding job in this guide configures a GitHub actions workflow for the newly created project.
```

**Review by jhaals — COMMENTED** [review]
Thanks a lot for improving the getting started guide! 🥇 

Suggesting that we add a bit more context to why the workflow scope is required.

**Review by jhaals — APPROVED** [review]

**Comment by jhaals** [comment]
Thanks @tim775!

**Comment by suuus** [comment]
Congrats on your first PR merged @tim775! Happy to count you as a contributor.

**Pull request merged**